packages/tocco-util/src/mockData/data/user_model.json

```diff
@@ -143,6 +143,13 @@
         "mandatory": false
       }
     },
+    {
+      "fieldName": "decimal",
+      "type": "decimal",
+      "validation": {
+        "mandatory": false
```

Comment:
have you compared this with an actual model? Isn't there meta information like prePointDigits and postPointDigits?
Because in a second step you could implement the sync validation